shared/kscan/src/androidMain/kotlin/org/ncgroup/kscan/BarcodeAnalyzer.kt

```
@@ -102,6 +70,36 @@ class BarcodeAnalyzer(
+                imageProxy.close()
+            }.addOnCanceledListener {
+                onCanceled()
+                imageProxy.close()
+            }
+    }
+
+    private fun scanInverted(imageProxy: ImageProxy) {
+        val invertedImage =
+            try {
+                createInvertedInputImage(imageProxy)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)